docs/configuration.md

```diff
@@ -1,20 +1,20 @@
 ## LSP Configuration
 
-There are some tunable options and settings for `nil`.
+There are some tunable options and settings for nil.
```

Comment:
:thinking: So we don't need to quote `nil`?

Reply:
if the quoted snippet is meant to be copy-and-pasted, or is a piece of code that already exists and you're referring to it, backticks makes sense. if it's used as a name, i prefer to use double-quotes whenever ambiguity arises, otherwise no formatting is necessary imo.